client/components/jetpack-logo/index.jsx

```diff
@@ -65,7 +79,7 @@ const JetpackLogo = ( { full = false, monochrome = false, size = 32, className,
 		);
 	}
 
-	if ( 24 === size ) {
+	if ( 24 === size && shouldUseMonochromeLogo ) {
```

Comment:
Is what we get when using the following component expected to be the one before these changes?

```react
<JetpackLogo full={ false } monochrome={ false } size={ 24 } />
```

Reply:
`shouldUseMonochromeLogo` has already got the size condition in there. Feels a bit repetitive here. Or we could just use prop `monochrome`

Reply:
This part of the code is updated as per our discussions in the Slack thread